marshal.go

```diff
@@ -370,7 +422,7 @@ func (e *Encoder) encInt(i int) error {
 	} else if 0xffff < i && i <= 0xffffff {
 		len = 3
 	} else if 0xffffff < i && i <= 0x3fffffff {
-		//for compatibility with 32bit Ruby, Fixnum should be less than 1073741824
+		// for compatibility with 32bit Ruby, Fixnum should be less than 1073741824
```

Comment:
If we intend to PR back to the original repo, I'd suggest we don't make whitespace changes.